src/dag-node.js

```diff
@@ -82,6 +82,9 @@ module.exports = class DAGNode {
 
   // addNodeLink - adds a DAGLink to this node that points to node by a name
   addNodeLink (name, node) {
+    if (!name) {
```

Comment:
Let me clarify:

`''`, `new Buffer()` - good :+1: -- let it through

`null`, `[]`, `undefined` - bad :-1:  -- fail and return an error